xonotic.patch

```diff
@@ -96,13 +97,13 @@ index 50d829e..85fbe91 100644
 +
 +.PHONY: all-zip-source
 +all-zip-source:
-+	( cd source/d0_blind_id && ./configure --enable-static --disable-shared $(RIJNDAELDETECT_CONFIGURE) )
-+	$(MAKE) -C source/d0_blind_id
++	( cd source/d0_blind_id && ./configure --prefix /app --enable-shared --enable-static $(RIJNDAELDETECT_CONFIGURE) )
++	$(MAKE) -C source/d0_blind_id all install
 +	$(MAKE) -C source/gmqcc
 +	$(MAKE) -C source/qcsrc QCC=$(CURDIR)/source/gmqcc/gmqcc
-+	$(MAKE) -C source/darkplaces sv-release DP_CRYPTO_STATIC_LIBDIR=$(CURDIR)/source/d0_blind_id/.libs
-+	$(MAKE) -C source/darkplaces cl-release DP_CRYPTO_STATIC_LIBDIR=$(CURDIR)/source/d0_blind_id/.libs
-+	$(MAKE) -C source/darkplaces sdl-release DP_CRYPTO_STATIC_LIBDIR=$(CURDIR)/source/d0_blind_id/.libs
++	$(MAKE) -C source/darkplaces sv-release LIB_CRYPTO="-lgmp -L/app/lib -ld0_blind_id"
```

Comment:
Same here. Untested on my side, but I think we could probably change the lib path to `-L$(PREFIX)/lib`.

Reply:
Last thing I do not like. Will merge the PR after this is addressed and I test the new build. :-)

Reply:
Done, you're free to test